basics/create-first-project.md

```diff
@@ -17,3 +26,77 @@ here you can set the **name** of your project and you can add some optional info
 This data will be used to forward any unmocked requests directly to your API service. _This data is confidential and never leaves your computer_.
 
 Clicking on **Add Project** the project will be created.
+
+## Mock Server
+
+Once the project is created you can see on the **Status Bar** that the server is working and its url; this is the base url used by all endpoints you will create.
```

Comment:
People don't really understands semi-colons. Let's use a normal `.` instead of `;`.